src/main/java/com/cloudogu/smeagol/repository/domain/RepositoryId.java

```diff
@@ -0,0 +1,57 @@
+package com.cloudogu.smeagol.repository.domain;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import com.google.common.base.Strings;
+
+import java.util.Objects;
+
+/**
+ * Unique id of a repository.
+ */
+public class RepositoryId {
+
+    private final String value;
+
+    private RepositoryId(String value) {
+        this.value = value;
+    }
+
+    /**
+     * Returns the string representation of the repository id.
+     *
+     * @return string representation
+     */
+    public String getValue() {
+        return value;
+    }
+
+    /**
+     * Creates a repository id from a string.
+     *
+     * @param value string a name
+     *
+     * @return name
+     */
+    public static RepositoryId valueOf(String value) {
+        checkArgument(!Strings.isNullOrEmpty(value), "repository id can not be null or empty");
+        return new RepositoryId(value);
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS00122)